docs/GettingStarted.md

```diff
@@ -1,19 +1,22 @@
 
 ## The SnappyData Approach
-At SnappyData, a very different approach is taken. SnappyData fuses a low latency, highly available in-memory transactional database (GemFireXD) into Spark with shared memory management and optimizations. Data in the highly available in-memory store is laid out using the same columnar format as Spark. All query engine operators are more optimized through better vectorization and code generation. The net effect is, an order of magnitude performance improvement when compared to native Spark caching, and more than two orders of magnitude better Spark performance when working with external data sources.
+SnappyData fuses a low latency, highly available in-memory transactional database (GemFire) into Spark with shared memory management and optimizations. Data in the highly available in-memory store is laid out using the same columnar format as Spark. Query engine operators are optimized through better vectorization and code generation. The net effect of these changes are, an order of magnitude performance improvement when compared to native Spark caching, and more than two orders of magnitude better Spark performance when working with external data sources.
 
-Essentially, Spark is turned into an in-memory operational database capable of transactions, point reads, writes, working with Streams (Spark) and running analytic SQL queries.
+Essentially, Spark is turned into an in-memory operational database capable of transactions, point reads, writes, working with Streams and running analytic SQL queries.
 
 ![SnappyData Architecture](Images/SnappyArchitecture.png)
 
-Conceptually, you can think of SnappyData as an **in-memory database that uses Spark's API and SQL as its interface and computational engine**- to process streams, work with myriad data sources like HDFS, and process data through a rich set of higher level abstractions. While the SnappyData engine is primarily designed for SQL processing, applications can work with Objects through Spark RDDs and the newly introduced Spark Datasets.
+SnappyData is an in-memory database that runs Spark’s compute engine directly in the database, and offers **Spark's API and SQL as its interface and computational engine**. The fusion with Spark allows SnappyData to work with a large number of data sources like HDFS, NoSQL etc. through bespoke Spark connectors. </br>While the SnappyData engine is primarily designed for SQL processing, applications can also work with Objects through Spark RDDs and the Spark Datasets API.
 
```

Comment:
SnappyData engine => add: SnappyData engine **(that builds on Spark Catalyst SQL engine)**